Readme.md

```diff
@@ -0,0 +1,55 @@
+﻿# Anuket Documentation
+
+Welcome to the Anuket Project Documentation repository. This repository contains the source files for https://docs.anuket.io/
+
+## Installation
+
+To get started with the Anuket documentation locally, clone this repository:
+
+```bash
+git clone https://github.com/anuket-project/Anuket-documentation.git
+cd Anuket-documentation
+```
+
+## Install the required dependencies:
+
+### Verify Python and Pip Installation
+
+Ensure Python and pip are installed and accessible in your environment.
+
+```bash
+python --version
+pip --version
+```
+
+### Install Sphinx

```

Comment:
Sphinx and all other dependencies will be installed by tox. 

Here we should install only tox with `pip install tox`